Sources/Commands/Options.swift

```diff
@@ -328,6 +328,15 @@ public struct SwiftToolOptions: ParsableArguments {
         help: "Specify the .netrc file path.",
         completion: .file())
     var netrcFilePath: AbsolutePath?
+    
+#if canImport(Security)
```

Comment:
If we conditionalize the existence of this option, passing it on unsupported platforms would cause an error. So I think we might need to do something like this:

```swift
#if canImport(Security)
    /// Whether to use keychain for authenticating with remote servers
    /// when downloading binary artifacts or communicating with a registry.
    @Flag(inversion: .prefixedNo,
          exclusivity: .exclusive,
          help: "Search credentials in OS keychain")
    var keychain: Bool = true
#else
    /// Whether to use keychain for authenticating with remote servers
    /// when downloading binary artifacts or communicating with a registry.
    @Flag(inversion: .prefixedNo,
          exclusivity: .exclusive,
          help: .hidden)
    var keychain: Bool = false
#end
```

Something like that would swallow any inapplicable keychain flags. And then in the validation method, we could do another check to emit a warning or error.